config/log.conf

```diff
@@ -8,7 +8,7 @@ log_format main_json escape=json
     '"body_bytes_sent":"$body_bytes_sent",'
     '"host":"$host",'
     '"http_connection":"$http_connection",'
-    '"http_referrer":"$http_referer",'
+    '"http_referer":"$http_referer",'
```

Comment:
From [the HTTP referer Wikipedia page](https://en.wikipedia.org/wiki/HTTP_referer) ...

> The misspelling of referrer was introduced in the original proposal by computer scientist Phillip Hallam-Baker to incorporate the field into the HTTP specification. The misspelling was set in stone by the time of its incorporation into the Request for Comments standards document RFC 1945; document co-author Roy Fielding has remarked that neither "referrer" nor the misspelling "referer" were recognized by the standard Unix spell checker of the period.